tests/acl/test_acl.py

```diff
@@ -536,6 +546,33 @@ def create_or_remove_acl_table(duthost, acl_table_config, setup, op, topo):
     time.sleep(30)
 
 
+def check_create_msg_in_syslog(duthost, ssh_hdl, log_msg):
+    """
+    Checks for a given log message after the last start-LogAnalyzer message in syslog
+
+    Args:
+        duthosts: All DUTs belong to the testbed.
+        ssh_hdl: SSH handle to the DUT
+        log_msg: Log message to be searched
+
```

Comment:
move it to some utility API's and also see if we can use exisitng `SSHClient` class

Reply:
Done @abdosi